js-intro.js

```diff
@@ -84,12 +107,14 @@ var testString = "thisisateststring"
 var toonimals = [ {name: "Itchy", animal: "mouse"}, {name: "Stimpy", animal: "cat"}, {name: "Daffy", animal: "duck"}, {name: "Scratchy", animal: "cat"}, {name: "Ren", animal: "dog"}, {name: "Felix", animal: "cat"}]
 
 // 5a. Write a function that takes in the toonimals array as an argument and returns an array with only the toon objects that are cats. Expected output: [ { name: "Stimpy", animal: "cat" }, { name: "Scratchy", animal: "cat" }, { name: "Felix", animal: "cat" } ]
-
+var  onlyCats = toonimals.filter((value) => value.animal === "cat")
+console.log(onlyCats)
 
 
 
 //5b. Write a function that returns only the names of all the non-cats. Expected output: "Itchy" "Daffy" "Ren"
-
+var nonCats = toonimals.map((value) => value.name)
+console.log(nonCats)
 
```

Comment:
var nonCats = toonimals.filter(value => {
  return value.animal !== 'cat'})
console.log(nonCats)
This will get you the non-cats rather than all the names.